content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -240,26 +225,21 @@ service succeeds after the RBAC policy has become effective.
 
     When the http header includes a valid JWT with the `groups` claim
     value [`"group1"`, `"group2"`]
-    containing `group1`, it should return HTTP code 200:
-
-    {{< text plain >}}
-    200
-    {{< /text >}}
+    containing `group1`, it should return HTTP code 200.
```

Comment:
Why is this sentence written in the conditional? All other texts regarding output were not. I suggest:
The HTTP header including a valid JWT with the `groups` claim value of [`"group1"`, `"group2"`] returns HTTP code 200 since it contains `group1`

Reply:
Done.